.gitignore

```diff
@@ -28,3 +28,4 @@ nuget.exe
 project.lock.json
 .build/
 .testPublish/
+launchSettings.json
```

Comment:
We're really inconsistent about this. Some of the samples require customized launch settings (e.g. Security). I'd rather check them in everywhere.